lib/puppy.rb

```diff
@@ -0,0 +1,18 @@
+# lib/puppy.rb
+
```

Comment:
please remove un needed or commented out lines. Your code should be clean and clear enough where other users should be able to tell what is going on.